src/server.rs

```diff
@@ -52,27 +34,29 @@ use std::collections::HashMap;
 use std::fmt;
 use std::time;
 use std::collections::VecDeque;
-use futures::{Future, Sink, Stream, stream, future::ok, sync::mpsc};
-use tokio_core::reactor::{Core, Timeout};
-use rand::RngCore;
-use semver::Version;
-use chrono::Local;
-use log::LevelFilter;
-use clap::{App, Arg};
-
-const TICK_INTERVAL_IN_MS:    u64      = 10;
-const NETWORK_INTERVAL_IN_MS: u64      = 100;    // Arbitrarily chosen
-const HEARTBEAT_INTERVAL_IN_MS: u64    = 1000;    // Arbitrarily chosen
-const MAX_ROOM_NAME:          usize    = 16;
-const MAX_NUM_CHAT_MESSAGES:  usize    = 128;
-const MAX_AGE_CHAT_MESSAGES:  usize    = 60*5; // seconds
-const SERVER_ID:              PlayerID = PlayerID(u64::max_value()); // 0xFFFF....FFFF
+
+
+use crate::tokio_core::reactor::{Core, Timeout};
+use crate::chrono::Local;
+use crate::log::LevelFilter;
+use crate::futures::{Future, Sink, Stream, stream, future::ok, sync::mpsc};
+use crate::rand::RngCore;
+use crate::semver::Version;
+use crate::clap::{App, Arg};
+
+pub const TICK_INTERVAL_IN_MS:    u64      = 10;
```

Comment:
Any particular reason to make the stuff in `server.rs` public? We will only have one server, right? I was thinking this `server.rs` could be our permanent server program, rather than being imported elsewhere.

Reply:
Was thinking about if we ever wanted a gui-based server controller.
[See an example here](https://imgur.com/qDJplm8)

I'll undo it for now since this wasn't a complicated refactor.